curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/step-060.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-60
 
 # --description--
 
-You can add a <dfn>fallback</dfn> value for the font-family by adding another font name separated by a comma. This second font would be used in case the browser does not have the font built-in to it.
+You can add a <dfn>fallback</dfn> value for the font-family by adding another font name separated by a comma. This second font would be used in case the browser does not have the font built into it.
```

Comment:
Second sentence is still a bit weird for me. Is it clear, that _second font_ and _the font_ are two different fonts?

Reply:
I think it is clear, although you could change "...does not have the font.." to "...does not have your primary font...". 

Also, I actually had notes on whether fonts are "built" or "installed" into browsers. But it comes down to what would a user thinks, or even cares about when it comes to browsers and fonts.

Reply:
I second this: Looking back at this, it is my understanding that the fallback can be any other font, provided the browser (page) has access to it - it does not have to be a font from _within_ the browser.

As such, I suggest we change the whole sentence to:
```suggestion
You can add a <dfn>fallback</dfn> value for the font-family by adding another font name separated by a comma. Fallbacks are used in instances where the initial is not found/available.
```

Reply:
Would you want to add the word "font" in between your phrase "initial is"? And if that is okay, do I just replace the sentence that is in there now with your sentence? And then git add and a commit notating the change and do the command I normally would: `git push origin fix/campercafe-typos`?

Reply:
@Kernix13 The subject ("font") is implicit, but I left it out because, in CSS, you can have fallbacks for things other than fonts as well.

You can just use the GitHub UI to `Commit suggestion`, if you want.

Thank you.